module-3/backend/week_one.md

```diff
@@ -4,8 +4,8 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 
 Note: When you're done, submit a PR. 
 
-1. What is `json` and why is it important?
-2. What's the difference between `joins` and `includes` in ActiveRecord?
-3. What's an API?
-4. How do we test an internal API (in general)?
-5. What are two different ways to customize your `json`?
+1. What is `json` and why is it important? JavaScript Object Notation: it's a lightweight and easy to use way of formatting data
+2. What's the difference between `joins` and `includes` in ActiveRecord? Joins allows you to specify which attributes you want from the associated table, while includes automatically loads them all.
+3. What's an API? Application Program Interface: it's a wrapper that allows users to interact with our data in specified ways, or vice versa
```

Comment:
i don't know if i'd call an API a wrapper. i'd need to hear more about what you mean by this!